### DISCUSSION TITLE
Using specific themes for specific file types

### DISCUSSION THREAD

**Discussion by ghost** [sharkdp/bat#2598] [Q&A] [answered]
Is there an inbuilt way to configure which theme is used for specific file types? I'm using a custom theme that doesn't provide syntax highlighting for certain file types so I want to use a different theme as a fallback. 
I've looked into this quite a bit but haven't found any answers.

**Comment by keith-hall** [comment, marked as answer]
Hi, no, `bat` currently doesn't have any built-in support for that.
Probably the best it can do is with a wrapper script which invokes `bat` and points to a different config file based on file extension. Obviously this would be limited to those files with specific extensions, it wouldn't work for piping from stdin for example.

**Reply by ghost** [reply]
Ah okay, I don't really have a lot of scripting knowledge so I'm going to have to look into that.
Thanks for the response!
